spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
+  describe 'registers user' do
```

Comment:
Please change to http method and method name instead

Reply:
👍 In this case should be `describe 'POST #create'` do

Reply:
Please fix the other places as well.